File: Iniciante/URI 1035.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var line = lines.shift().split(" ");
var A = parseInt(line[0]);
var B = parseInt(line[1]);
var C = parseInt(line[2]);
var D = parseInt(line[3]);

if ((B > C) && (D > A) && ((C+D) > (A+B)) && (C > 0) && (D > 0) && (A % 2 == 0)) console.log('Valores aceitos'); 
else console.log('Valores nao aceitos');

File: Iniciante/URI 1035.rb
line = gets.split(' ')
a = line[0].to_i
b = line[1].to_i
c = line[2].to_i
d = line[3].to_i

if b > c && d > a && c > 0 && d > 0 && (c + d) > (a + b) && a % 2 == 0
	puts "Valores aceitos"
else 
	puts "Valores nao aceitos"
end